src/NServiceBus.Core/IEndpointInstanceExtensions.cs

```diff
@@ -15,6 +16,7 @@ public static class IEndpointInstanceExtensions
         /// </summary>
         /// <param name="endpoint">The endpoint to stop.</param>
         /// <param name="gracefulStopTimeout">The length of time granted to gracefully complete processing.</param>
+        [SuppressMessage("Code", "PCR0019:A task-returning method should have a CancellationToken parameter or a parameter implementing ICancellableContext", Justification = "Convenience method wrapping the CancellationToken overload.")]
```

Comment:
This right here proves that an analyzer is better than the reflection tests. You'd never get that level of understanding from trying to read through the reflection tests 6 months from now.